src/main/resources/templates/error/5xx.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" data-th-replace="~{base :: layout(~{::title}, ~{::main})}">
+<head>
+  <title>Server Error</title>
+  <link rel="stylesheet" href="../../static/style.css"/>
+</head>
+<body>
+<main class="error500">
+  <h1 class="error500-title">
+    500 Server Error
```

Comment:
This was originally written just to handle 500 errors, as `error/500.html`, but I incompletely refactored it. This should be generalized to handle all 5xx errors.

Reply:
Force-pushed to fix.